shared/parser.go

```diff
@@ -350,7 +351,8 @@ func Parse(argList []string) ParserResult {
 		"-fopenmp":       {0, pr.compileLinkUnaryCallback},
 
 		"-Wl,-dead_strip": {0, pr.warningLinkUnaryCallback},
-		"-dead_strip":     {0, pr.warningLinkUnaryCallback}, //iam: tor does this. We lose the bitcode :-(
+		"-dead_strip":     {0, pr.warningLinkUnaryCallback},  //iam: tor does this. We lose the bitcode :-(
+		"-target":         {1, pr.compileLinkBinaryCallback}, //drpdr: cross-compiling, e.g. CHERI
```

Comment:
Let's not add our usernames in.

Reply:
You are right. I kept the upstream's convention. I'll remove it.